satpy/tests/reader_tests/test_mviri_l1b_fiduceo_nc.py

```diff
@@ -36,13 +37,16 @@
     DatasetWrapper,
     FiduceoMviriEasyFcdrFileHandler,
     FiduceoMviriFullFcdrFileHandler,
+    Interpolator,
 )
 from satpy.tests.utils import make_dataid
 
 # NOTE:
 # The following fixtures are not defined in this file, but are used and injected by Pytest:
 # - request
 
+fill_val = int("4294967295")
```

Comment:
Why is it defined as string and then converted to int?

Reply:
this was done as a temporary bugfix but can be changed back (done)

Reply:
I now remember more clearly- after pushing to Github, a unit test fails for windows 3.9 (overflow error), from web search I found out that this minor change resolves the issue.

Reply:
Ok, can you please add a comment :)